tutorials/BatchProcessingSequenceMap.ipynb

```diff
@@ -0,0 +1,629 @@
+    "session = onnxruntime.InferenceSession('resize_and_crop_image.onnx', None)\n",
+    "out_images = []\n",
+    "for image in images:\n",
+    "    result = session.run([], {'image': np.array(image)})\n",
+    "    out_images.append(Image.fromarray(result[0]))\n",
+    "show_images(out_images)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f96563fe",
+   "metadata": {},
+   "source": [
+    "We are now ready to use the graph we defined together with SequenceMap, to perform batch processing. A non-uniform batch of samples in ONNX can be represented by a sequence of tensors, where each tensor can have a different shape.\n",
+    "\n",
+    "SequenceMap operator applies a subgraph (body) to each tensor in the input sequence, producing an output sequence of processed samples. Let us build a model that utilizes SequenceMap to apply the previously created graph to a sequence:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "4733e8a4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "resize_and_crop_batch = helper.make_model(\n",
+    "    onnx.helper.make_graph(\n",
+    "        nodes=[\n",
+    "            onnx.helper.make_node('SequenceMap', body=resize_and_crop, inputs=['batch'], \n",
+    "                                  outputs=['batch_processed'])\n",
+    "        ],\n",
+    "        name='resize_and_crop_batch',\n",
+    "        inputs=[onnx.helper.make_tensor_sequence_value_info('batch', onnx.TensorProto.UINT8, ['H', 'W', 'C'])],\n",
+    "        outputs=[onnx.helper.make_tensor_sequence_value_info('batch_processed', onnx.TensorProto.UINT8, \n",
+    "                                                             ['H', 'W', 'C'])],\n",
+    "    ),\n",
+    "    producer_name='test',\n",
+    "    opset_imports=[op]\n",
+    ")\n",
+    "checker.check_model(resize_and_crop_batch)\n",
+    "onnx.save(resize_and_crop_batch, 'resize_and_crop_batch.onnx')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "93c4737d",
+   "metadata": {},
+   "source": [
+    "With the model saved, we can now create a session and feed several images to a single run call, as a list of numpy array:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "06995904",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Output sizes: \n",
+      "(224, 224)\n",
+      "(224, 224)\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2022-03-29 13:25:49.485994387 [W:onnxruntime:, constant_folding.cc:202 ApplyImpl] Unsupported output type of N11onnxruntime22SequenceTensorTypeBaseE. Can't constant fold SequenceEmpty node '0x3df93000x3e25280'\n",
+      "2022-03-29 13:25:49.486736036 [W:onnxruntime:, constant_folding.cc:202 ApplyImpl] Unsupported output type of N11onnxruntime22SequenceTensorTypeBaseE. Can't constant fold SequenceEmpty node '0x3df93000x3e25280'\n"
```

Comment:
Let me understand more context about it: regarding to the ONNXRuntime warning here: has this batch processing been fully supported by ONNXRuntime? If so, is it possible to resolve the warning here?

Reply:
`SequenceMap` operator is currently on review (I'll move this PR to a draft PR until it gets merged).
That being said, the warning comes from `SequenceEmpty` operator, which is fully supported by ONNXRuntime. I've brought this up here: https://github.com/microsoft/onnxruntime/issues/11041. 

The issue seems to be about ONNX Runtime not being able to constant fold a sequence output. In case of sequences there is no initializer field in the GraphProto that could be used to store the constant. Only tensors and sparse tensors are supported now. I believe that we could extend the GraphProto to cover sequences as well. In the meantime, @hariharans29 suggested they could lower the severity of the log for the case of sequences.